spec/features/contestants_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe "Contestants index", type: :feature do
+    describe 'contestants#index' do
+        it 'shows a list of all names of all the contestants and a list of projets they have been on' do
+            jay = Contestant.create(name: "Jay McCarroll", age: 40, hometown: "LA", years_of_experience: 13)
+            gretchen = Contestant.create(name: "Gretchen Jones", age: 36, hometown: "NYC", years_of_experience: 12)
+            kentaro = Contestant.create(name: "Kentaro Kameyama", age: 30, hometown: "Boston", years_of_experience: 8)
+            erin = Contestant.create(name: "Erin Robertson", age: 44, hometown: "Denver", years_of_experience: 15)
+            
+            recycled_material_challenge = Challenge.create(theme: "Recycled Material", project_budget: 1000)
+            furniture_challenge = Challenge.create(theme: "Apartment Furnishings", project_budget: 1000)
+
+            news_chic = recycled_material_challenge.projects.create(name: "News Chic", material: "Newspaper")
+            boardfit = recycled_material_challenge.projects.create(name: "Boardfit", material: "Cardboard Boxes")
+
+            upholstery_tux = furniture_challenge.projects.create(name: "Upholstery Tuxedo", material: "Couch")
+            lit_fit = furniture_challenge.projects.create(name: "Litfit", material: "Lamp")
+
+            ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
+            ContestantProject.create(contestant_id: gretchen.id, project_id: news_chic.id)
+            ContestantProject.create(contestant_id: gretchen.id, project_id: upholstery_tux.id)
+            ContestantProject.create(contestant_id: kentaro.id, project_id: upholstery_tux.id)
+            ContestantProject.create(contestant_id: kentaro.id, project_id: boardfit.id)
+            ContestantProject.create(contestant_id: erin.id, project_id: boardfit.id)
+            
+            # When I visit the contestants index page ("/contestants")
+            visit "/contestants"
+            # I see a list of names of all the contestants
+            # # And under each contestants name I see a list of the projects (names) that they've been on
+            Contestant.all.each do |contestant|
+                # Expect contestant name
+                expect(page).to have_content(contestant.name)
+
+                # Expect projects list after name
+                projects = @contestant.projects.map(&:name).join(", ")
+                expect(page).to have_content("#{@contestant.name} - Projects: #{projects}")
+            end
+            # expect(page).to have_content(jay.name)
```

Comment:
You were actually on the right track with listing out each name, but you don't necessarily need to have as much test data as you do. You could stick to 2-3 contestants, and each one has 1-2 projects it's on. 

Once you make the projects display in a comma separated list, then for each contestant, you could have one assertion for its project. You could also add text in your view so that you have the contestant name in the label so you know which contestant the projects are for (i.e. `<p><%= "Projects for  #{contestant.name} are: #{contestant.project_list}" %> </p>`)

expect(page).to have_content("Projects for Jay are: News Chic")

expect(page).to have_content("Projects for Gretchen are: News Chic, Upholstery Tuxedo")